api/search_test.go

```diff
@@ -48,97 +52,96 @@ func TestSearchHandlerFunc(t *testing.T) {
 		So(resp.Body.String(), ShouldContainSubstring, "Invalid offset parameter")
 		So(qbMock.BuildSearchQueryCalls(), ShouldHaveLength, 0)
 		So(esMock.MultiSearchCalls(), ShouldHaveLength, 0)
+		fmt.Println("It's lunch time")
```

Comment:
This made me laugh - I assume this shouldn't be here

Reply:
😱